packages/hash/api/src/seed-data/dev-users.ts

```diff
@@ -43,13 +46,43 @@ export const ensureDevUsersAreSeeded = async ({
 }): Promise<UserModel[]> => {
   const createdUsers = [];
 
-  for (const { email, shortname, preferredName, isInstanceAdmin } of devUsers) {
+  // Only use `devUsers` if we are in a dev environment
+  let usersToSeed = isDevEnv ? devUsers : [];
+
+  // Or if we're explicitly setting users to seed.
+  if (process.env.HASH_SEED_USERS) {
+    try {
+      usersToSeed = JSON.parse(
+        process.env.HASH_SEED_USERS,
+      ) as DevelopmentUser[];
+    } catch (error) {
+      logger.error(
+        "Could not parse environment variable `HASH_SEED_USERS` as JSON. Make sure it's formatted correctly.",
+      );
+    }
+  }
+
+  for (let index = 0; index < usersToSeed.length; index++) {
+    const {
+      email,
+      shortname,
+      preferredName,
+      password = devDefaultPassword,
```

Comment:
```suggestion
      password = "password",
```

? Not sure I see value in the new var given that it’s used only in one place.

Reply:
Good point, I suppose we get around having to name the variable if we inline it :+1:

Reply:
Addressed in 1bf5eba04529c98b28c0822abd5a765e96a05091